### PR TITLE
Under the Twin Suns: the S'rand'marr Sector

### PR DESCRIPTION
**Pull request by Alberyk**
This pr adds the S'rand'marr sector. It includes a new skybox and a skybox Adhomai object. It comes with visitable lore exoplanets. Including space ruins made especially for Adhomai and Raskara.

Stuff that still needs to be done:
- [x] stargazer icon
- [x] adhomai animals sprites
- [x] some other minor sprite assets
- [x] more space ruins for the exoplanets

### CODE REVIEW

File: code/game/objects/structures/hadii_statue.dm [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+		health -= damage
+	if(health <= 0)
+		topple()
+
+/obj/structure/hadii_statue/bullet_act(var/obj/item/projectile/Proj)
+	if(toppled)
+		return
+	if(!Proj)
+		return
+	do_integrity_check(Proj.damage)
```
Comment (NonQueueingMatt) [inline code/game/objects/structures/hadii_statue.dm:71]: there should be a damage amount check here, like the one above in attackby

File: code/modules/mob/living/simple_animal/hostile/pra.dm [hunk 2]
```diff
@@ -125,3 +127,115 @@
...
+
+	response_help = "pets"
+	response_disarm = "gently pushes aside"
+	response_harm = "hits"
+	a_intent = I_HURT
+	stop_automated_movement_when_pulled = 0
+
+	speak_emote = list("beeps")
+	emote_hear = list("buzzes","beeps")
+	speak = list("Hadii's grace, comrades.","The stars belongs to the People's Republic of Adhoomai!", "The quest for knowledge must continue!")
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/simple_animal/hostile/pra.dm:153]:
```suggestion
	speak = list("Hadii's grace, comrades.","The stars belong to the People's Republic of Adhoomai!", "The quest for knowledge must continue!")
```

**Review by NonQueueingMatt — APPROVED** [review]

**Review by alsoandanswer — APPROVED** [review]

**Comment by SleepyGemmy** [comment]
DNM while i review.

File: code/game/objects/structures/bonfire.dm [hunk 3]
```diff
@@ -0,0 +1,388 @@
...
+			if(600 to 900)
+				heat_range = 4
+			if(900 to 1300)
+				heat_range = 5
+			if(1300 to 2000)
+				heat_range = 6
+	else
+		heat_range = 0
+	return heat_range
+
```
Comment (SleepyGemmy) [inline code/game/objects/structures/bonfire.dm:158]:
```suggestion
```

**Review by SleepyGemmy — APPROVED** [review]

File: code/modules/clothing/head/xenos/tajara.dm [hunk 4]
```diff
@@ -131,6 +136,12 @@
 	icon_state = "nka_consularhat"
 	item_state = "nka_consularhat"
 
+/obj/item/clothing/head/tajaran/archeologist
+	name = "archeologist hat"
```
Comment (SleepyGemmy) [inline code/modules/clothing/head/xenos/tajara.dm:140]:
```suggestion
	name = "archaeologist hat"
```

**Pull request merged**